src/models/components/policy/tahh_normal_stochastic_policy.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import torch.distributions as distributions
+import torch.nn as nn
+import torchrl
+from torch import Tensor
+from torch.distributions.distribution import Distribution
+
+from .stochastic_policy import StochasticPolicy
+
```

Comment:
エントロピー計算の参照を実装に入れておいてください。PPOの更新で使います。
```suggestion
from torchrl.modules import TanhNormal as _TanhNormal

class TanhNormal(_TanhNormal):
    def entropy(self):
        self.base_dist.entropy()
```

Reply:
対応しました https://github.com/MLShukai/PrimitiveAMI/pull/16/commits/0904e98281aea93ddfdab97f690661310ee8bcc1